src/main/java/com/cs3365/rpgdapi/weapons/WeaponsController.java

```diff
@@ -47,4 +48,15 @@ public WeaponsController(WeaponsService weaponsService) {
         return ResponseEntity.created(location)
                              .body("");
     }
+    
+    @RequestMapping(value = "/files/{id}")
```

Comment:
You can replace this with `@GetMapping("/{id}")`. It will append your route to the top level route we have. Also, when creating resources for a REST API you'd want this resource to be `/weapons/{weaponId}`. With the way it is now, it'll show up as `/weapons/files/{id}`. The naming convention we're trying to follow here is `/resourceName/{uniqueResourceIdentifier}`. Does that make sense?

Reply:
Yes, that makes sense